ff/src/fields/models/fp/montgomery_backend.rs

```diff
@@ -368,7 +368,7 @@ pub trait MontConfig<const N: usize>: 'static + Sync + Send + Sized {
     }
 
     #[inline]
-    #[unroll_for_loops(12)]
+    #[cfg_attr(not(target_family = "wasm"), unroll_for_loops(12))]
```

Comment:
```suggestion
    #[cfg_attr(not(target_family = "wasm"), unroll_for_loops(12))]
    #[cfg_attr(target_family = "wasm", unroll_for_loops(6))]
```

This way we can still have some unrolling for `wasm`

Reply:
@hrxi if you could confirm this compiles, that would be great =)

Reply:
Still compiles.